subprojects/com.mbeddr/platform/build.gradle

```diff
@@ -55,7 +55,6 @@ def bundledDeps = [
             exclude module: 'commons-compress'
             exclude group: 'org.bouncycastle'
             exclude module: 'xmlsec'
-            exclude module: 'batik-all'
```

Comment:
Why did you remove this exclude for the transitive dependency? Batik stuff is not bundled and not used in apache-poi anyway. It actually won't change anything since the jars would just be downloaded, but not used. So for consistency I would prefer to also keep the dependency excluded here.

Reply:
tbh. I just removed than, cause I though: mbeddr is no longer dealing with batik, so let's ditch it.